src/python/pants/backend/helm/subsystems/unittest.py

```diff
@@ -42,12 +42,18 @@ class HelmUnitTestSubsystem(ExternalHelmPlugin):
         "macos_x86_64": "macos-amd64",
     }
 
+    color = BoolOption(
+        "--color", default=False, help="Enforce printing colored output even stdout is not a tty."
+    )
+
     output_type = EnumOption(
         "--output-type",
         default=HelmUnitTestReportFormat.XUNIT,
-        help="Output type used for the test report",
+        help="Output type used for the test report.",
     )
 
+    strict = BoolOption("--strict", default=None, help="Strict parse the test suites.")
```

Comment:
Should probably unify-with and reference the help for the field.

But also: now that #15809 has landed, you probably don't need to support setting a default on a subsystem? cc @kaos

Reply:
Agreed. No need for "global" default field settings in subsystems any more.
Simply adding this to a root `BUILD` file would have the same effect:
```python
__defaults__({helm_unittest_test: dict(strict=<bool>)})
```